sw/device/lib/testing/kmac_testutils.c

```diff
@@ -0,0 +1,66 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/kmac_testutils.h"
+
+#include "sw/device/lib/dif/dif_kmac.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+
+void kmac_testutils_config(dif_kmac_t *kmac, bool sideload) {
```

Comment:
nit: I recommend to either to use a name that incorporates the fact that this function uses software configured entropy, or add a `dif_kmac_entropy_mode_t` parameter to the function to make this more explicit on the call site.